tests/Google/Ads/GoogleAds/Lib/V1/GoogleAdsClientTest.php

```diff
@@ -85,6 +88,10 @@ public function testGetClientOptions()
             GoogleAdsLoggingInterceptor::class,
             $clientOptions['transportConfig']['grpc']['interceptors'][0]
         );
+        $this->assertSame(
+            getenv('http_proxy'),
```

Comment:
Should we test "https_proxy" too?

Reply:
"https_proxy" is actually not needed, i removed it.